org.testeditor.tsl.dsl/src/main/java/org/testeditor/tsl/dsl/Tsl.xtext

```diff
@@ -17,9 +17,9 @@ import "http://www.testeditor.org/tsl"
 
 TslModel:
 	{TslModel}
-	('package' package=QualifiedName )?
-	(NL+ ML_COMMENT | NL+ SL_COMMENT)*
-	(NL+ specification=TestSpecification)?
+	(NL* 'package' package=QualifiedName)?
```

Comment:
This allows the `TestSpecification` to start in the same line as the package declaration, right? This would be valid:
```
package org.testeditor.my.package #MyFirstTestSpec
```
(Do we want that?)

Conversely, this does *not* allow the `TestSpecification` to start right after a multi-line comment, i.e. I can't write
```
/* This is a multi-line comment */ #MyFirstTestSpec
```
Again, not sure if that's an issue or not…

Reply:
changed the first, kept the second. the name of the specification should always start the line.